packages/widgetbook/lib/src/fields/double_field.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+
+import 'field.dart';
+import 'field_codec.dart';
+import 'field_type.dart';
+
+class DoubleField extends Field<double> {
+  DoubleField({
+    required super.group,
+    required super.name,
+    super.initialValue = 0,
+    required this.min,
+    required this.max,
+    this.divisions,
+    required super.onChanged,
+  }) : super(
+          type: FieldType.double,
+          codec: FieldCodec(
+            toParam: (value) => value.toString(),
+            toValue: (param) => double.tryParse(param ?? ''),
+          ),
+        );
+
```

Comment:
Yeah, it will be too confusing. What if we go with factory constructors or as an alternative provide different Fields e.g. `DoubleSliderField`, `DoubleInputField`, `DoubleStepperField` (stepper will be with a number label and + and - button to increase and decrease the value (could also be part of the `DoubleInputField`)

Reply:
After implementing this, I see that will not have an ideal knob naming.

| Field               | Type     | Knob                      |
|---------------------|----------|---------------------------|
| `DoubleField`*      | `double` | `knobs.double(...)`       |
| `NumberField`*      | `num`    | `knobs.number(...)`       |
| `DoubleSliderField` | `double` | `knobs.doubleSlider(...)` |
| `DoubleInputField`  | `double` | `knobs.doubleInput(...)`  |

\* current implementation